tutorials/private-forseti-with-scc-integration/index.md

```diff
@@ -123,14 +122,14 @@ In Cloud Console ensure you have the `forseti` project selected, then do the fol
 
 Note: This is also a good time to ensure the database is in the same zone as the server VM (optional).
 
-4. Navigate to "VCP Network > Firewall Rules" page and edit the rules of both `forseti-client-allow-ssh-external` and `forseti-server-allow-ssh-external` to restrict the `Source IP Ranges` to that of the locaton (on-premise) you will be connecting from:
+4. Navigate to "VCP Network > Firewall Rules" page and edit the rules of both `forseti-client-allow-ssh-external` and `forseti-server-allow-ssh-external` to restrict the `Source IP Ranges` to that of the locaton (on-premise or datacenter range) you will be connecting from:
 
 ![58b996ae.png](58b996ae.png)
 
 5. (Optional) Navigate to the "Compute Engine" > "VM Instances" page and remove the public IP of both the server and client VMs.
-   - This will require you to access the instances through IAP proxy or NAT gateway
+   - This will require you to access the instances through IAP proxy or VPN as well as Interconnect
 
-6. (Optional) Set up a [NAT gatway](https://cloud.google.com/nat/docs/using-nat#gce-setup) for internet access.
+6. (Optional)  More details on how to setup [IAP](https://cloud.google.com/iap/docs/using-tcp-forwarding#ssh_tunneling) to access forseti VM through SSH from internet.
```

Comment:
still need to keep nat if they want external internet access (maybe call it out more clearly) and add IAP

So step 6 optional IAP, Step 7 optional NAT